scilpy/tracking/tracker.py

```diff
@@ -486,7 +486,7 @@ class GPUTacker():
     mask : ndarray
         Tracking mask. Tracking stops outside the mask.
     seeds : ndarray (n_seeds, 3)
-        Seed positions in voxel space with origin `corner`.
+        Seed positions in voxel space with origin ` center`.
```

Comment:
Why all these corner -> center modifications ?

Reply:
+ extra space

Reply:
Conversion to corner is now done inside `GPUTracker` instead of  `main`, so that the same input seeds are passed to DIPY and GPU.